src/dialect/postgres/postgres-driver.ts

```diff
@@ -112,8 +122,38 @@ class PostgresConnection implements DatabaseConnection {
     }
   }
 
-  async *executeQueryStream<R>(): AsyncIterableIterator<QueryResult<R>> {
-    throw new Error('@TODO: postgres::executeQueryStream')
+  async *executeQueryStream<O>(
+    compiledQuery: CompiledQuery,
+    chunkSize: number
+  ): AsyncIterableIterator<QueryResult<O>> {
+    if (!this.#options.cursorIpl) {
+      throw new Error(
+        "'cursorIpl' is not present is postgres dialect config! It's required for streaming for work in postgresql!"
```

Comment:
```suggestion
        "'cursorImpl' is not present in your postgres dialect config! It's required for streaming for work in postgresql!"
```

I'll see myself out :P

No but seriously, nice work. I'm also wondering about `Ipl` vs `Impl`. Are the various uses of `Ipl` throughout this change a typo?

Personally, I would prefer `cursor` instead of `cursorImpl` from a configuration standpoint for consistancy with `pool`, but no big deal either way

Reply:
> No but seriously, nice work. I'm also wondering about Ipl vs Impl. Are the various uses of Ipl throughout this change a typo?

Actually, I intended to use `Impl`. Good catch!

> Personally, I would prefer cursor instead of cursorImpl from a configuration standpoint for consistancy with pool, but no big deal either way

@koskimas what's your opinion on this one?

Reply:
I prefer just `cursor`. Also I think it should take the constructor itself and not a function. That returns an instance. Simply `cursor: Cursor`